vllmd-hypervisor/generate-runtime-image.sh

```diff
@@ -0,0 +1,593 @@
+        echo "ERROR: cloud-hypervisor-v44 is not installed. Please install it first."
+        missing=1
+    fi
+    
+    # Check for mkdosfs (part of dosfstools)
+    if ! command -v /usr/sbin/mkdosfs &>/dev/null; then
+        echo "ERROR: /usr/sbin/mkdosfs not found. Please install dosfstools package."
+        missing=1
+    fi
+    
+    # Check for mcopy (part of mtools)
+    if ! command -v /usr/bin/mcopy &>/dev/null; then
+        echo "ERROR: /usr/bin/mcopy not found. Please install mtools package."
+        missing=1
+    fi
+    
+    # Check for truncate (part of coreutils)
+    if ! command -v truncate &>/dev/null; then
+        echo "ERROR: truncate not found. Please install coreutils package."
+        missing=1
+    fi
+    
+    # Check for curl
+    if ! command -v /usr/bin/curl &>/dev/null; then
+        echo "ERROR: /usr/bin/curl not found. Please install curl package."
+        missing=1
+    fi
+    
+    # Check for IP tool
+    if ! command -v /usr/sbin/ip &>/dev/null && ! command -v /sbin/ip &>/dev/null; then
+        echo "ERROR: ip command not found. Please install iproute2 package."
+        missing=1
+    fi
+    
+    # Check for iptables
+    if ! command -v /usr/sbin/iptables &>/dev/null && ! command -v /sbin/iptables &>/dev/null; then
+        echo "ERROR: iptables not found. Please install iptables package."
+        missing=1
+    fi
+    
+    # Check for preseed file
+    if [[ ! -f "${PRESEED_PATH}" ]]; then
+        echo "ERROR: Preseed file not found at ${PRESEED_PATH}"
+        missing=1
+    fi
+    
+    if [[ "${missing}" -eq 1 ]]; then
+        exit 1
+    fi
+}
+
+# Check for existing files
+check_existing_files() {
+    local existing=0
+    
+    if [[ -f "${OUTPUT_PATH}" ]]; then
+        echo "WARNING: Output image already exists at ${OUTPUT_PATH}"
+        existing=1
+    fi
+    
+    if [[ "${existing}" -eq 1 && "${FORCE}" -ne 1 ]]; then
+        echo "Use --force to overwrite existing files"
+        exit 1
+    fi
+}
+
+# Create necessary directories
+create_directories() {
+    if [[ ! -d "${STATE_DIR}" ]]; then
+        if [[ "${DRY_RUN}" -eq 0 ]]; then
+            mkdir -p "${STATE_DIR}"
+            echo "Created state directory: ${STATE_DIR}"
+        else
+            echo "[DRY RUN] Would create state directory: ${STATE_DIR}"
+        fi
+    fi
+    
+    # Create build directory for temporary files with timestamp
+    BUILD_DIR="${STATE_DIR}/${TIMESTAMP}-build"
+    if [[ ! -d "${BUILD_DIR}" ]]; then
+        if [[ "${DRY_RUN}" -eq 0 ]]; then
+            mkdir -p "${BUILD_DIR}"
+            echo "Created build directory: ${BUILD_DIR}"
+        else
+            echo "[DRY RUN] Would create build directory: ${BUILD_DIR}"
+        fi
+    fi
+}
+
+# Download Debian netinst ISO
+download_installer() {
+    local iso_url="https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-12.9.0-amd64-netinst.iso"
+    local iso_path="${BUILD_DIR}/debian-netinst.iso"
```

Comment:
change BUILD_DIR to BUILD_PATH. change iso_path to iso_filepath. make this change for every variable